### PR TITLE
8.0 crm claim stage to crm claim type dev osval

### PR DESCRIPTION
**Pull request by osvalr**
As part of this module, stages based on claim types have been added. there is a demo video below that shows this.

[CRM Claim Types](http://www.youtube.com/watch?v=jcKjHXvFb2k)

### CODE REVIEW

**Comment by osvalr** [comment]
@yaniaular can you review this PR? this is was commented out in https://github.com/OCA/crm/pull/39#issuecomment-130415339

**Comment by yaniaular** [comment]
@osvalr :+1:

File: crm_claim_type/README.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 CRM Claim Types
 ===============
 
-Includes a way to classify claims adding the concept of type.
+Includes a way to classify claims adding the concept of type and with this it allows to configure claim stages depending on claim types.
```
Comment (nhomar) [inline crm_claim_type/README.rst]: Please add Use cases discussed here.

File: crm_claim_type/__openerp__.py [hunk 2]
```diff
@@ -23,18 +23,24 @@
     'name': 'CRM Claim Types',
     'category': 'Customer Relationship Management',
     'summary': 'Claim types for CRM',
-    'author': 'Odoo Community Association (OCA),'
+    'author': 'Odoo Community Association (OCA), '
```
Comment (nhomar) [inline crm_claim_type/__openerp__.py]: This White space remove it please-

File: crm_claim_type/README.rst (hunk 1 above)
Comment (osvalr) [inline crm_claim_type/README.rst]: @yaniaular what use cases must be included? Would it be just fine with some examples?

Comment (yaniaular) [inline crm_claim_type/README.rst]: @osvalr add the classify claims by default on this module: Customer, Supplier and Other.

**Comment by yaniaular** [comment]
@nhomar can you approve this PR?

**Comment by yaniaular** [comment]
@osvalr can you make rebase in this PR?

**Comment by zaoral** [comment]
+1

**Pull request merged**